src/lib/ecore/ecore_time.c

```diff
@@ -61,7 +61,7 @@ ecore_time_unix_get(void)
    gettimeofday(&timev, NULL);
    return (double)timev.tv_sec + (((double)timev.tv_usec) / 1000000);
 #else
-# error "Your platform isn't supported yet"
+# warning "Your platform isn't supported yet"
```

Comment:
why is this change needed?

Reply:
I maybe got it wrong but wasn't your [suggestion](https://github.com/expertisesolutions/efl/pull/294#discussion_r453602013)?

Reply:
for while `gettimeofday` wasn't being implemented, yes. Now that we have it and we define `HAVE_GETTIMEOFDAY` there is no reason to maintain this change.